packages/synapse-react-client/src/components/CreateTableViewWizard/CreateTableViewWizard.integration.test.tsx

```diff
@@ -70,6 +70,25 @@ async function getLatestCreatedColumnModelIdsFromSpy(
   ).list.map(cm => cm.id)
 }
 
+function useGetFeatureFlagsOverride() {
```

Comment:
Can you put a reusable version of this in `mocks/msw/handlers/featureFlagHandlers.ts`, then use that in this file?